ui/.eslintrc.json

```diff
@@ -13,7 +13,8 @@
     "jest": true
   },
   "rules": {
-    "no-console": "warn"
+    "no-console": "warn",
+    "@typescript-eslint/explicit-module-boundary-types": "off"
```

Comment:
can we define return type to exported function instead of defining rule here ?I guess will work.